src/parser/core/modules/DISPLAY_ORDER.ts

```diff
@@ -5,8 +5,9 @@ enum DISPLAY_ORDER {
 	SUGGESTIONS = -20,
 	STATISTICS = -10,
 	// Module DO is 0-100
-	TIMELINE = 110,
-	CHANGELOG = 120,
+	COMBOS = 110,
```

Comment:
The whole point of me putting them 10 apart was so you _didn't_ need to renumber them, dork.
Coulda put it in as 105 lel

Reply:
I did that... then I looked at it and went :ablobtriggered: because everything else was 10 apart...